Historical Tweet Search.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[45]:
+
+
+import twint
+import pandas as pd
+import numpy as np
+import nest_asyncio
+nest_asyncio.apply()
+
+# list of search areas
+searches = [["40.676302, -73.971787, 1mi","BK"],["40.588579, -74.143985, 3mi","SI"]]
+
+# storing each search into csv file
+for search in searches:
+    c = twint.Config()
+    c.Store_csv = True
+    c.Geo = search[0]
+    c.Output = "historical_search_"+search[1]+".csv"
+    c.Search = "blm OR \"black lives matter\" OR police OR protest OR riot OR \"breonna taylor\" OR \"george floyd\"" #"blm" #search terms
+    #c.Min_likes = 20
```

Comment:
did you comment it out bc there was something wrong w/ that config or just for your own testing??

Reply:
i was originally trying different search terms so that's why there are those comments, i will remove them to avoid confusion